01_Data/src/layers/sequelize/repository/Base.ts

```diff
@@ -19,14 +19,30 @@ export class SequelizeRepository<T extends Model<any, any>> extends CrudReposito
     this.namespace = namespace;
   }
 
-  async readByKey(key: string): Promise<T | undefined> {
+  async readByKey(key: string | number): Promise<T | undefined> {
     return await this.s.models[this.namespace].findByPk(key).then((row) => row as T);
   }
 
   async readAllByQuery(query: object): Promise<T[]> {
     return await this.s.models[this.namespace].findAll(query as FindOptions<any>).then((row) => row as T[]);
   }
 
+  async readAllBySqlString(sqlString: string): Promise<object[]> {
```

Comment:
Is this something we should consider adding to CrudRepository as well? Or will the syntax be too sequelize-specific?